acquia/mc-cs-plugin-custom-objects
# Adaptation to Mautic v5 - Autowiring + deprecations

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 


:warning: This PR needs the changes from [#308](https://github.com/acquia/mc-cs-plugin-custom-objects/pull/308), [#316](https://github.com/acquia/mc-cs-plugin-custom-objects/pull/316), so they should be put together for testing.

#### Description:


This PR adds autowiring and fixes some deprecations.

#### Steps to test this PR:


1. Setup a custom object. Use Mautic v5.
2. Make sure everything is working.
